input/postgres/statements.go

```diff
@@ -186,9 +186,9 @@ func GetStatements(ctx context.Context, server *state.Server, logger *util.Logge
 			server.SelfTest.MarkCollectionAspectWarning(state.CollectionAspectPgStatStatements, "monitoring user may have insufficient permissions to capture all queries")
 			server.SelfTest.HintCollectionAspect(state.CollectionAspectPgStatStatements, "Please set up"+
 				" the monitoring helper functions (%s)"+
-				" or connect as superuser to get query statistics for all roles.", selftest.URLPrinter.Sprint("https://github.com/pganalyze/collector#setting-up-a-restricted-monitoring-user"))
+				" or connect as superuser to get query statistics for all roles.", selftest.URLPrinter.Sprint("https://pganalyze.com/docs/install/aiven/03_create_pg_stat_statements_helpers"))
```

Comment:
This is currently only covered in the Aiven docs, since Aiven is the only provider we support that does not grant `pg_monitor`. Should we add a separate section for this in the docs?